src/app/views/sidebar/resource-explorer/ResourceExplorer.tsx

```diff
@@ -226,11 +227,13 @@ const UnstyledResourceExplorer = (props: any) => {
         </>
       }
 
-      <Label styles={{ root: { position: 'relative', left: '10px' } }}>
+      { items[0].links.length > 0 && <Label styles={{ root: { position: 'relative', left: '10px' } }}>
```

Comment:
I think we should check if items and their content are defined before accessing them
`items && items.length > 0 && items[0].links  && items[0].links.length > 0`?